### PR TITLE
table : create component table event

### PR DESCRIPTION
**Pull request by turrs**
fitur create table component

### CODE REVIEW

File: src/Utils/Rating/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Rate } from 'antd'
+const Rating = (rating) => {
+  return (
```
Comment (argorfc) [inline src/Utils/Rating/index.js]: use jsx format extesion (.jsx)

File: src/Utils/Rating/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <div><Rate disabled allowHalf defaultValue={0} value={rating}></Rate></div>
```
Comment (argorfc) [inline src/Utils/Rating/index.js]: Please, use Rate component below div

File: src/components/TableData/table.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.ant-table-thead .ant-table-cell {
```
Comment (argorfc) [inline src/components/TableData/table.css]: Please watch the text indent

**Review by argorfc — CHANGES_REQUESTED** [review]
Not approved.
Please fix the comments!

File: src/Utils/Rating/index.js (hunk 1 above)
Comment (turrs) [reply src/Utils/Rating/index.js]: thankyou for your fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a

File: src/Utils/Rating/index.js (hunk 2 above)
Comment (turrs) [reply src/Utils/Rating/index.js]: thankyou for the fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a

File: src/components/TableData/table.css (hunk 3 above)
Comment (turrs) [reply src/components/TableData/table.css]: thankyou for the fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a

**Review by argorfc — APPROVED** [review]
Approved.

**Pull request merged**